docs/core/diagnostics/dotnet-stack.md

```diff
@@ -0,0 +1,116 @@
+---
+title: dotnet-stack tool - .NET Core
+description: Installing and using the dotnet-stack command-line tool.
+ms.date: 11/21/2019
+---
+# dotnet-stack performance analysis utility
+
+**This article applies to:** ✔️ .NET Core 3.0 SDK and later versions
+
+## Install dotnet-stack
+
+### Option 1 - Install as dotnet global tool:
```

Comment:
This part should be changed to match what https://github.com/dotnet/docs/pull/21617 did.